pipeline/stream.go

```diff
@@ -2,6 +2,7 @@ package pipeline
 
 import (
 	"sync"
+	"sync/atomic"
```

Comment:
could this be a mistake, and uber/atomic is intended?

Reply:
Correct, I'll replace with uber lib.